src/TodoListItem.module.css

```diff
@@ -0,0 +1,104 @@
+:root {
```

Comment:
One of the cool things about css modules and React is that you can give each Component its own `MyComponent.module.css`. Then when you import `styles` into the matching component it will only ever import the styles you specifically set in the module, and no other styles can accidentally be applied bc they have overlapping selectors for `input` or something else generic. 

As you work toward your final project, see if you can break up this single css module into 1 for each of your components that you want to style!